cmd/root.go

```diff
@@ -57,27 +59,31 @@ Edit this once you have created your source
 			os.Exit(1)
 		}
 
-		region := viper.GetString("aws-region")
+		regions := viper.GetString("aws-regions")
```

Comment:
There's a method in viper called [GetStringSlice](https://pkg.go.dev/github.com/spf13/viper#GetStringSlice) which likely makes more sense here since it does the splitting for you. Plus would support a config file and other potential future things

Reply:
@DavidS-om Why'd you decide to go with Unmarshall rather than StringSlice?

Reply:
results with `GetStringSlice`:
```
INFO[0000] Got config                                    auto-config=false aws-access-key-id=AKIA3VLV2U27SACUIUPF aws-regions="[eu-west-2,us-west-2,ap-southeast-2]" aws-secret-access-key="[REDACTED]" max-parallel=80 nats-jwt= nats-name-prefix= nats-nkey-seed= nats-servers="[nats://localhost:4222 nats://nats:4222]"
FATA[0001] Error retrieving account information          error="operation error STS: GetCallerIdentity, exceeded maximum number of attempts, 3, https response error StatusCode: 0, RequestID: , request send failed, Post \"https://sts.eu-west-2,us-west-2,ap-southeast-2.amazonaws.com/\": dial tcp: lookup sts.eu-west-2,us-west-2,ap-southeast-2.amazonaws.com: no such host"
exit status 1
```

UnmarshalKey is how viper documents CSV values: https://sagikazarmark.hu/blog/decoding-custom-formats-with-viper/

The downside afaict is that this wouldn't work with a config file (or require the csv string in the config file. The above blog also shows examples to build a full decoder that reflects on the incoming type, which could address the issue of different formats (types (string vs []string)).

Reply:
Oh nice. Adding that to the fact that I've just found this: https://github.com/spf13/viper/issues/380 which still isn't fixed I'd agree Unmarshall is the way to go, especially since you've documented that it should be comma-separated